source/administration/devops-command-center.rst

```diff
@@ -0,0 +1,268 @@
+
+When an incident has started and the incident channel is created, the channel header is updated to reflect its origin and
+a message from the incident bot is also posted to the incident channel naming the creator of the incident. If an incident is started from the context of a post, the channel header includes a link to the post and the text of the post is also posted to the channel alongside a permalink.
+
+Incidents are displayed in the RHS are visible to all members of the relevant team, even if they are not a member of the corresponding incident channel. It is not possible to view incidents from teams other than the currently selected team.
+
+Using Playbooks
+~~~~~~~~~~~~~~~~
+
+Playbooks are templates which are applied to an incident and define a set of steps to be followed in order to resolve the incident. Playbooks can be refined over a period of time to match the changing parameters of future related incidents.
+
+The set of steps contained within a Playbook is called a checklist.
+
+Viewing and creating Playbooks
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Select **! Incidents** from the plugin menu in the channel header to open the RHS panel. Select the Playbooks icon to open the Playbooks backstage and list all Playbooks associated with the current team. Select **+ New Playbook** to start a new Playbook and checklist. Enter a Playbook name and choose **Save Playbook**.
+
+Creating checklists
+~~~~~~~~~~~~~~~~~~~
+
+Checklists are created within a Playbook and provide steps to follow during an incident. Checklists can be created ahead of time as part of a Playbook, or during an incident.
+
+
+Editing checklists
+~~~~~~~~~~~~~~~~~~~
+
+You can add steps and edit checklist items during an active incident. You can also edit checklist items in Playbooks that are not being used in active incidents. 
+
+To edit a checklist, choose **Edit**.
+
+- Select the text field to edit the item details. 
+- Hover over an input box and select the **X** to remove the checklist item.
+- Drag the hamburger menu to rearrange the checklist items.
+
+Choose **Done** to save the changes.
+
+
+Ending incidents
+~~~~~~~~~~~~~~~~
+
+There are two ways to end an incident:
+
+- Using slash command */incident end* from within the incident channel.
+- Using the **End Incident** button in the RHS panel while in the incident channel.
+
+The incident will become inactive, be removed from the list of active incidents, and the associated channel will be archived.
+
+Administrator's Guide
+^^^^^^^^^^^^^^^^^^^^^^
+
+Permissions
+~~~~~~~~~~~~~~~~~~~~~
+
+Incidents and Playbooks are associated with teams in Mattermost, and permissions are assigned based on the Permission scheme used in Mattermost. Incidents are directly tied to a specific team and channel.
+
+Public incidents
```

Comment:
Note that public/private isn't actually part of the release v0.4. Apologies for how the design doc tackles this, since it's a mixture of current and upcoming functionality. Not sure if we want to focus on v0.4 functionality only, where all incident channels are private?

Reply:
Gotcha. This doc should be aligned with the current release so this section can be removed until 0.5.